types/types.go

```diff
@@ -160,3 +161,48 @@ func (m ComMatrix) Contains(cd ComDetails) bool {
 
 	return false
 }
+
+func ToNFTables(m ComMatrix) ([]byte, error) {
+	var tcpPorts []string
+	var udpPorts []string
+	for _, line := range m.Matrix {
+		if line.Protocol == "TCP" {
+			tcpPorts = append(tcpPorts, fmt.Sprint(line.Port))
+		} else if line.Protocol == "UDP" {
+			udpPorts = append(udpPorts, fmt.Sprint(line.Port))
+		}
+	}
+
+	tcpPortsStr := strings.Join(tcpPorts, ", ")
+	udpPortsStr := strings.Join(udpPorts, ", ")
+
+	result := fmt.Sprintf(`#!/usr/sbin/nft -f
+
+	table ip filter {
+		chain FIREWALL {
+			# Allow loopback traffic
+			type filter hook input priority 0; policy accept;
```

Comment:
Have you tried applying the generated nft output in your nftables?
It won't work, invalid syntax. Remove this line.

Here is a simple example:

```
table ip filter {
        chain FIREWALL {
                type filter hook input priority 0; policy accept;
        }
        chain INPUT {
                type filter hook input priority 0; policy accept;
                jump FIREWALL
        }
}
```


```
communication-matrix-master.nft:8:8-15: Error: Could not process rule: Operation not supported
		jump FIREWALL
		     ^^^^^^^^
```